src/solver/simulation/solver.h

```diff
@@ -183,6 +192,8 @@ class ISimulation : public Impl
     std::shared_ptr<Yuni::Job::QueueService> pQueueService = nullptr;
     //! Result writer
     Antares::Solver::IResultWriter::Ptr pResultWriter = nullptr;
+
+    RANDOM_ARRAYS randomArrays;
```

Comment:
I see this was introduced but it any usage in this PR. Did I miss something ?

Reply:
From my understanding, this is work in progress

Reply:
Needs to be used instead of the extern var, it's a WIP